website/content/docs/upgrading/deduplication/different-case.mdx

```diff
@@ -0,0 +1,184 @@
+due to unrelated changes in storage such as when deleting related entities or
+groups.
+
+</Warning>
+
+When different-case variations of an alias name actually represent different
+logical users or entities in an external system, investigate each alias before
+resolving duplicates. You **must** resolve the duplicates on each mount before
+enabling forced identity deduplication.
+
+**If you confirm you need to keep the duplicated aliases**, you will need to
+reconfigure the authN mount or external service to stop creating case-sensitive
+aliases. For example, you could append a unique ID to the alias name to
+differentiate between similar usernames. Be mindful of how the modified behavior
+may disrupt users as you roll out the change.
+
+<Note>
+
+We are not aware of specific integrations where changing mount or external
+service behaviors is necessary and expect those situations to be rare. If you
+determine your deployment requires changes to authN mounts or external services,
+we encourage Vault Enterprise customers to work with HashiCorp support staff to
+determine the best strategy and implementation.
+
+</Note>
+
+**If you can confirm you do not need the duplicated alias, or the entire auth
+mount**, delete all but one of the entity aliases on each mount using the ID
+noted in the log line with `id=`.
+
+<Tabs>
+
+<Tab heading="CLI" group="cli">
+
+Use `vault delete` with the `/identity/entity-alias/id/{id}` path to delete the
+duplicate identity:
+
+```shell-session
+$ vault delete /identity/entity-alias/id/<id>
+```
+
+For example, if one of the duplicate entity log entries includes
+`id=df3568a4-3b65-4104-9481-1129ecbed72f`:
+
+<CodeBlockConfig hideClipboard="true">
+
+```shell-session
+$ vault delete /identity/entity-alias/id/df3568a4-3b65-4104-9481-1129ecbed72f
+
+Success! Data deleted (if it existed) at: identity/entity-alias/id/df3568a4-3b65-4104-9481-1129ecbed72f
+```
+
+</CodeBlockConfig>
+
+</Tab>
+
+<Tab heading="API" group="api">
+
+Call the [`/identity/entity-alias/id/{id}`](/vault/api-docs/secret/identity/entity-alias#delete-entity-alias-by-id)
+endpoint to delete the duplicate entry:
+
+```shell-session
+$ curl                                      \
+   --request DELETE                         \
+   --header "X-Vault-Token: ${VAULT_TOKEN}" \
+   ${VAULT_ADDR}/v1/identity/entity-alias/id/<id>
+```
+
+For example, if one of the duplicate entity log entries includes
+`id=df3568a4-3b65-4104-9481-1129ecbed72f`:
+
+<CodeBlockConfig hideClipboard="true">
+
+```shell-session
+$ curl                                      \
+   --request DELETE                         \
+   --header "X-Vault-Token: ${VAULT_TOKEN}" \
+   ${VAULT_ADDR}/v1/identity/entity-alias/id/df3568a4-3b65-4104-9481-1129ecbed72f | jq
+```
+
+</CodeBlockConfig>
+
+</Tab>
+
+</Tabs>
+
+
+After you resolve the duplicates by deleting them or reconfiguring the relevant
+services, recheck the system logs to confirm the duplicates no longer appear in
+the log during unseal.
```

Comment:
```suggestion
the log during the next unseal.
```
Not sure if we should spell out - you might need to explicitly restart a standby node to observe this or wait for it to happen naturally?